### PR TITLE
[hotfix][doc] fix code mismatch in filesystem.md

### PR DESCRIPTION
**Pull request by zjuwangg**

### CODE REVIEW

File: docs/content/docs/connectors/datastream/filesystem.md [hunk 1]
```diff
@@ -315,9 +315,9 @@ final FileSink<String> sink = FileSink
     .forRowFormat(new Path(outputPath), new SimpleStringEncoder<String>("UTF-8"))
     .withRollingPolicy(
         DefaultRollingPolicy.builder()
-            .withRolloverInterval(Duration.ofSeconds(10))
-            .withInactivityInterval(Duration.ofSeconds(10))
-            .withMaxPartSize(MemorySize.ofMebiBytes(1))
+            .withRolloverInterval(Duration.ofMinutes(15))
+            .withInactivityInterval(Duration.ofMinutes(5))
+            .withMaxPartSize(MemorySize.ofMebiBytes(1024))
```
Comment (MartijnVisser) [inline docs/content/docs/connectors/datastream/filesystem.md:320]: For me it's not clear why these values need to be changed. Can you elaborate on that?

**Review by MartijnVisser — COMMENTED** [review]
I'm not sure why you think there's a mismatch, looking forward to your thoughts on that.

File: docs/content/docs/connectors/datastream/filesystem.md (hunk 1 above)
Comment (zjuwangg) [reply docs/content/docs/connectors/datastream/filesystem.md:320]: Oh yes. If you look this article line 357, 
```
This example creates a simple sink that assigns records to the default one hour time buckets. It also specifies a rolling policy that rolls the in-progress part file on any of the following 3 conditions:
 - It contains at least 15 minutes worth of data
 - It hasn't received new records for the last 5 minutes	
 - The file size has reached 1 GB (after writing the last record)	 
```
 you will find the description is not matched with the code arguments here.
Thx for your review.

Comment (MartijnVisser) [reply docs/content/docs/connectors/datastream/filesystem.md:320]: Thanks, I missed that.

**Review by MartijnVisser — APPROVED** [review]
LGTM

**Pull request merged**